data/data/bootstrap/baremetal/files/etc/ironic.env.template

```diff
@@ -7,3 +7,5 @@ DHCP_ALLOW_MACS="{{.PlatformData.BareMetal.ProvisioningDHCPAllowList}}"
 # Used by ironic to allow ssh to running IPA instances
 IRONIC_RAMDISK_SSH_KEY="{{.SSHKey}}"
 IPTABLES={{if .PlatformData.BareMetal.ProvisioningIPv6}}ip6tables{{else}}iptables{{end}}
+# Used by IPA to configure additional NTP servers with chrony
+ADDITIONAL_NTP_SERVERS="{{.PlatformData.BareMetal.AdditionalNTPServers}}"
```

Comment:
The type of `AdditionalNTPServers` here is `[]string`, so we are relying on golang templating to convert this implicitly into the format we are expecting. Does it? (But also, let's not.)

Reply:
fixed